src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -18,6 +18,8 @@ public static partial class Certificates
         {
             private const string CertificatePassword = "testcertificate";
             private const string TestDataFolder = "TestData";
+            private static Mutex m = new Mutex(false, "Global\\CoreFXTest.Configuration.Certificates.LoadPfxCertificate");
```

Comment:
> So applying this mutex at a lower level (product-level instead of test) isn't feasible to work around the OS bug?

@bartonjs would know more details about this decision on the .Net side.